angular.json

```diff
@@ -1,131 +1,138 @@
 {
-    "$schema": "./node_modules/@angular/cli/lib/config/schema.json",
-    "version": 1,
-    "newProjectRoot": "projects",
-    "projects": {
-        "bookshelf-app": {
-            "projectType": "application",
-            "schematics": {
-                "@schematics/angular:component": {
-                    "style": "scss"
-                }
+  "$schema": "./node_modules/@angular/cli/lib/config/schema.json",
```

Comment:
What have you exactly changed here?

Reply:
The file was rewritten with different whitespacing formatting during postinstall after I run `npm add @angular/fire`. VS code version control did not show the styling changes (only code changes), so I was not aware of them.

Reply:
Ok